DigitalLearningSolutions.Web/ViewModels/MyAccount/NotificationPreferencesViewModel.cs

```diff
@@ -1,15 +1,28 @@
 ﻿namespace DigitalLearningSolutions.Web.ViewModels.MyAccount
 {
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models;
+
     public class NotificationPreferencesViewModel
     {
-        public NotificationPreferencesViewModel(int? adminId, int? delegateId)
+        public NotificationPreferencesViewModel(
+            int? adminId,
+            int? delegateId,
+            IEnumerable<NotificationPreference> adminNotifications,

```

Comment:
Should this be a list now (given the conversion to list in the controller)?

Reply:
I used that conversion to quiet down a multiple enumeration complaint, but the new logic doesn't have that issue so I left it all as Enumerable.